tests/e2e/idempotency/test_idempotency_dynamodb.py

```diff
@@ -22,12 +22,16 @@ def parallel_execution_handler_fn_arn(infrastructure: dict) -> str:
     return infrastructure.get("ParallelExecutionHandlerArn", "")
 
 
+@pytest.fixture
+def parallel_functions_handler_fn_arn(infrastructure: dict) -> str:
+    return infrastructure.get("ParallelFunctionsHandlerArn", "")
+
+
 @pytest.fixture
 def idempotency_table_name(infrastructure: dict) -> str:
     return infrastructure.get("DynamoDBTable", "")
 
 
-@pytest.mark.xdist_group(name="idempotency")
 def test_ttl_caching_expiration_idempotency(ttl_cache_expiration_handler_fn_arn: str):
```

Comment:
Recently @rubenfonseca worked to improve the e2e tests in this PR: https://github.com/awslabs/aws-lambda-powertools-python/pull/1861
I think you need to rebase/merge your local branch with the latest updates and add this decorator `@pytest.mark.xdist_group(name="XX")` before test function signature.